test/command/cmd.go

```diff
@@ -204,35 +214,53 @@ func (s *Starter) KillAll() {
 			continue
 		}
 		if v, ok := cmd.Value.(*exec.Cmd); ok {
-			s.kill(v)
+			err := s.kill(v)
+			if err != nil {
+				return err
+			}
 		}
 	}
+	return nil
 }
 
 // Clean cleans all temporary directories and processes.
-func (s *Starter) Clean() {
-	s.KillAll()
+func (s *Starter) Clean() error {
+	err := s.KillAll()
+	if err != nil {
+		return err
+	}
 	if s.Home != "" {
-		os.RemoveAll(s.Home)
+		err = os.RemoveAll(s.Home)
+		if err != nil {
+			return err
+		}
 	}
+	return nil
 }
 
 // kill can not be called directly, use Kill instead.
-func (s *Starter) kill(cmd *exec.Cmd) {
+func (s *Starter) kill(cmd *exec.Cmd) error {
 	if cmd == nil || cmd.Process == nil {
-		return
+		return nil
 	}
 	if cmd.ProcessState == nil {
-		cmd.Process.Signal(os.Interrupt)
+		err := cmd.Process.Signal(os.Interrupt)
```

Comment:
```
if err := cmd.Process.Signal(os.Interrupt); err != nil {
	return err
}
```

Could you please use the code style as above? THX.

Reply:
@Starnop 
I don't understand this.
It is different with above.

Reply:
I mean you can update your code style

from
```
err := cmd.Process.Signal(os.Interrupt)
if err != nil {
    return err
}
```

to
```
if err := cmd.Process.Signal(os.Interrupt); err != nil {
	return err
}
```

😄